postgres/problems/problem_09.md

```
@@ -1,21 +1,23 @@
-# 09 「A さんが買ったが B さんが買っていない」商品
+# 09 ユーザ別注文数ランキング
 
 **難易度**：★★  
-**学習トピック**：IN, 集合演算, サブクエリ
+**学習トピック**：SELECT, LEFT JOIN, サブクエリ, ORDER BY
 
 ## 説明
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)